src/main/resources/scripts/realistic drops.sk

```diff
@@ -1,14 +1,14 @@
-#
-# these triggers fix some drops.
-#
-
-on break of glass:
-	drop glass
-
-on break of glass pane:
-	drop glass pane
-
-on break of bookshelf:
-	cancel event
-	set block to air
-	drop bookshelf
+#
+# these triggers fix some drops.
+#
+
+on break of glass:
+	drop glass
+
+on break of glass pane:
+	drop glass pane
+
+on break of bookshelf:
+	cancel event
+	set block to air
+	drop bookshelf
```

Comment:
Remove unnecessarily large diff (Also if this remains colored glass support should be added)

Reply:
This large diff is caused by the rename of the file. About colored glass, I can add it of course!

Reply:
you didn't rename the file, it's just a line-ending issue

Reply:
Outdated issue + don't know how resolve a commit with a line-ending issue without add any ``.gitattribute`` file